model/DataQuery.php

```diff
@@ -779,6 +779,22 @@ public function selectFromTable($table, $fields) {
 	}
 
 	/**
+	 * Add the given fields from the given table to the select statement.
+	 *
+	 * @param String $table Unquoted table name (will be escaped automatically)
+	 * @param Array $fields Database column names (will be escaped automatically)
+	 */
+	public function addSelectFromTable($table, $fields) {
+		$fieldExpressions = array_map(function($item) use($table) {
+			return "\"$table\".\"$item\"";
```

Comment:
Can we please not add in new conventions into 3.x? All of our core code is ansi-92 sql standard, and a convention change (especially in a non-major branch) means we've lost all of our consistency.

You're also breaking all augmentSQL rewrites on db connectors that support multiple table quotation styles.

If you want to change this convention, it's worth doing as a part of an RFC and a complete work (rewrite of core) plus collecting feedback on impact of this change.